### ISSUE TITLE
empty label switch

### ISSUE THREAD

**Issue by jscharett** [US-CBP/cbp-theme#204]
Would be nice to have the switch be label aware, and remove the margin.  Something like
```
.switch {
    label:empty::before {
        margin-right: 0;
    }
}
```
Not sure of the accessibility issues around not having a label for a component, but would be nice to have a way of only showing the switch with no label.
